product_docs/docs/biganimal/release/overview/02_high_availability/distributed_highavailability.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "Distributed high availability"
+---
+
+Distributed high-availability clusters are powered by [EDB Postgres Distributed](/pgd/latest/). They use multi-master logical replication to deliver more advanced cluster management compared to a physical replication-based system. Distributed high-availability clusters let you deploy a cluster across multiple regions or a single region. For use cases where high availability across regions is a major concern, a cluster deployment with distributed high availability enabled can provide:
+
+-  Two data groups with a witness group in a third region
+-  Three data groups
```

Comment:
they don't provide 3 data groups. Can we change this sentence to 

"For use cases where high availability across regions is a major concern, a cluster deployment with distributed high availability enabled can provide two data groups with a witness group in a third region."